src/js/legend_config.js

```diff
@@ -35,6 +36,10 @@ export const sections = [
     name: "Route markers",
     source: "Wikidata",
   },
+  {
+    name: "Points of Interest",
```

Comment:
The legend uses Sentence case throughout. I realize this is a departure from many print maps, but since the legend is a floating panel separate from the map, I think it’s more relevant that sentence case is more common in UIs these days.

```suggestion
    name: "Points of interest",
```

If we want to use Title Case, then we’d need to install a library to intelligently title-case Wikidata labels, but to my knowledge such libraries only exist for English and would be inappropriate in other languages like Spanish and French anyways.